presentations/en/SLF.md

```diff
@@ -0,0 +1,47 @@
+# Senior Leadership Forum
+
+## IT Strategy - IITB
+
+---
+
+## Key Messages for the SLF presentation
+
+- What is the mandate of the IT Strategy team?
+- What are they currently working on?
+- What's in it for me?
+
+---
+
+## IT Strategy Mandate
+
+### Mission (reason for existence)
+
+Enable the strategic value of IT within ESDC to increase business agility.
+
+--
+
+### IT Stragy's mission context with Digital "Transformation"
+
+Change the relationship of Business & IT from a contracting model one *(a company within a company)*,
+to a partnership one *(shared accountability)*.
+
+--
+
+### Mandate (authority witin IITB)
+
+- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
+- Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks
+- Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
+- Challenge current thinking and approaches
+
+---
+
+## Team's current priorities
+
+- Be the catalysts of change within IITB to lay the path for the evolution of ESDC's digital services.
+
+---
+
+### What's in it for you
+
+- asdfasdfasdf
```

Comment:
? @remyb2canada

Reply:
the SLF.md file was accidently added as part of the PR.
It wasn't supposed to be

I need to stop using
`git add .`

:P